ucrop/src/main/java/com/yalantis/ucrop/UCropActivity.java

```diff
@@ -84,6 +85,8 @@
     private static final int ROTATE_WIDGET_SENSITIVITY_COEFFICIENT = 42;
 
     private String mToolbarTitle;
+    private int mGravity = Gravity.START;
+    private Float mToolbarTextSize = 14F;
```

Comment:
Please use some constant value instead of magic numbers? 
E.g. private static final float DEFAULT_TOOLBAR_TEXT_SIZE = 14F